engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/new/views/content-view-new.html

```diff
@@ -78,8 +78,7 @@ <h2 translate>Create Content View</h2>
                    ng-model="contentView.auto_publish"
                    type="checkbox"
                    tabindex="5"
-                   ng-disabled="!contentView.composite"
-                   ng-checked="contentView.composite && 0"/>
```

Comment:
for my own knowledge, what was the `&& 0` for?

Reply:
If contentView.composite is undefined (or false), this checkbox defaults to unchecked.

Reply:
Of course, this checkbox is bound to `contentView.auto_publish`, so there's no need to instantiate the checkbox state. It'll be whatever `contentView.auto_publish` is.